### ISSUE TITLE
Write Bill of Materials

### ISSUE THREAD

**Issue by gawlikaj** [gawlikaj/elek-labirynt#1]
Write the bill of materials for components used

**Comment by gawlikaj** [comment]
Initial Bill of Materials has been create, but part numbers will need to be assigned and more investigation will need to be done regarding pricing of a custom PCB, currently considering vendors that will be reasonably priced for a prototype board.  Considering contingencies if a prototype board is outside of price range.

**Comment by gawlikaj** [comment]
Draft of bill of materials has been completed, the rest of the bill of materials is contingent on completion of PCB design and cost of a custom PCB.  Starting new issue after PCB design draft is completed to investigate costs.
